ui/admin/app/components/form/onboarding/index.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable ember/no-empty-glimmer-component-classes */
+import Component from '@glimmer/component';
+
+export default class FormOnboardingComponent extends Component {}
```

Comment:
Why do we need an empty component class?

Reply:
Ouch! I forgot to commit last change in here, you are right @cameronperera we shouldn't have empty classes. I will work on it and provide an update.

Thanks for the suggestion 😉